Update README - Add Instructions To Upgrade SDK Version (#1401)
braintree/braintree_ios: README.md

Next, read the [**full documentation**](https://developer.paypal.com/braintree/docs/guides/payment-method-types-overview) for information about integrating with additional payment methods, such as PayPal and Venmo, as well as explore our pre-built [Drop-In UI offering](https://developer.paypal.com/braintree/docs/guides/drop-in/overview).

## Upgrade Your SDK Version

> If you're looking to update to a newer version of our SDK, please see our recommended approach below.

Reviewer: ```suggestion
If you're looking to update to a newer version of our SDK, please see our recommended approach below for each package manager. We always recommend updating to the latest version of the SDK which can be found in our [CHANGELOG](https://github.com/braintree/braintree_ios/blob/main/CHANGELOG.md).
```

### Using Swift Package Manager

>  To update using SPM, select _File→ Packages → Update to Latest Package Versions_.

Reviewer: ```suggestion
 To update using Swift Package Manager, select _File→ Packages → Update to Latest Package Versions_.
```

### Using Cocoapods 

> You can either update all pods listed within your Podfile using `pod update` or specific pods as needed using `pod update PODNAME`. For additional detail, see these [guidelines](https://guides.cocoapods.org/using/pod-install-vs-update.html).

Reviewer: ```suggestion
You can either update all pods listed within your Podfile using `pod update` or specific pods as needed using `pod update PODNAME`. For additional details, see the [Cocoapods guidelines](https://guides.cocoapods.org/using/pod-install-vs-update.html).
```

### Using Carthage

> To update to the latest versions of each framework, simply run the `carthage update` command. For more details, check out these [guidelines](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement).

Reviewer: ```suggestion
To update to the latest versions of each framework, simply run the `carthage update` command. For more details, check out the [Carthage guidelines](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement).
```
